### ISSUE TITLE
[BUG] Example 3 in chamfer_edge_mask apparently not working. 

### ISSUE THREAD

**Issue by RAMilewski** [revarbat/BOSL2#930]
**Describe the bug**
When I remove the # from the example code, I get this: 
![Screen Shot 2022-08-21 at 12 16 20 AM](https://user-images.githubusercontent.com/702294/185780133-aa04a7b7-0ea3-4304-8f85-49b1d4bd28e8.png)

**Code To Reproduce Bug**
```
include <BOSL2/std.scad>

diff("mask")
cube(50, center=true) {
    edge_mask(TOP+RIGHT)
        chamfer_edge_mask(l=50, chamfer=10);
}
	
```
**Expected behavior**
It should chamfer the edge. 

- OpenSCAD Version:   2021.01
- Other libraries used:  None.

**Comment by revarbat** [comment]
The mask tags changed a couple months back. Instead of `diff("mask")`, use `diff()` with no argument.

**Comment by revarbat** [comment]
The examples code *is* wrong, though.

**Comment by revarbat** [comment]
Fixes committed to PR #929
